modules/store/spec/runtime_checks.spec.ts

```diff
@@ -384,6 +400,39 @@ describe('Runtime checks:', () => {
   });
 });
 
+describe('ActionType uniqueness', () => {
+  beforeEach(() => {
+    resetRegisteredActionTypes();
+  });
+
+  it('should throw when having no duplicate action types', () => {
```

Comment:
shouldn't this be `'should throw when having duplicate action types'` ?

Reply:
Thanks!